ui/cfp-button.tsx

```diff
@@ -1,14 +1,25 @@
 import Link from "next/link";
 
 import styles from "./cfp-button.module.scss";
+import { ReactNode } from "react";
 
-function CfpButton() {
+interface CfpButtonProps {
+  url?: string;
+  children: ReactNode;
+}
+
+function CfpButton({url, children}: CfpButtonProps) {
     return (
         <div className={styles.main}>
-            <Link href="/" className={styles.button}>
-                Call for Papers
-            </Link>
-        </div>);
+          <button className={styles.button}>
```

Comment:
Could you style the next Link component as a button?

Wrapping links with <button> components is not a good practice (buttons are form/logical action elements, which may not suppose to navigate away from the page)

Reply:
that's a good catch. I will fix it, thx

Reply:
ff2779049b6130c02d6c8d6e0c8a4ab7a24e51e6